client/assets/stylesheets/shared/mixins/_elevation.scss

```diff
@@ -7,53 +7,53 @@
 	}
 	@else if $dp==1 {
 		box-shadow:
-			0 1px 1px 0 rgba(var(--color-neutral-100-rgb), 0.14),
-			0 2px 1px -1px rgba(var(--color-neutral-100-rgb), 0.12),
-			0 1px 3px 0 rgba(var(--color-neutral-100-rgb), 0.2);
+			0 1px 1px 0 color-mix(in srgb, var(--color-neutral-100) 14.000000000000002%, transparent),
```

Comment:
to fix

Reply:
😂

Reply:
Floating point math :smile: fixed in https://github.com/Automattic/wp-calypso/pull/101639/commits/986d0ebfd88b0b444907d773f440f9be0e8353c7